ui/client/components/graph/GraphPartialsInTS/drawGraph.ts

```diff
@@ -24,9 +23,9 @@ export function drawGraph(
   performance.mark("redrawing start")
   this.redrawing = true
 
-  const nodesWithGroups = NodeUtils.nodesFromProcess(process, expandedGroups)
-  const edgesWithGroups = NodeUtils.edgesFromProcess(process, expandedGroups)
-  const groups = NodeUtils.getExpandedGroups(process, expandedGroups)
+  const nodesWithGroups = NodeUtils.nodesFromProcess(process)
```

Comment:
This method should be named `nodesWithGroupsFromProcess` and below method also should be chaged.

Reply:
why, what we could get from that change?